[2021-07-16 18:13:26,791] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T18:13:01.008775+00:00 [queued]>
[2021-07-16 18:13:26,816] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T18:13:01.008775+00:00 [queued]>
[2021-07-16 18:13:26,817] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 18:13:26,818] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 18:13:26,819] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 18:13:26,834] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T18:13:01.008775+00:00
[2021-07-16 18:13:26,847] {standard_task_runner.py:52} INFO - Started process 12807 to run task
[2021-07-16 18:13:26,852] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T18:13:01.008775+00:00', '--job-id', '1680', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpsr8jqyje', '--error-file', '/tmp/tmpj98bwjc2']
[2021-07-16 18:13:26,853] {standard_task_runner.py:77} INFO - Job 1680: Subtask restart_test
[2021-07-16 18:13:26,910] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T18:13:01.008775+00:00 [running]> on host 13346afafa5b
[2021-07-16 18:13:26,981] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T18:13:01.008775+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T18:12:58.424428+00:00
[2021-07-16 18:13:28,414] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T181301, start_date=20210716T181326, end_date=20210716T181328
[2021-07-16 18:13:28,452] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 18:13:28,513] {local_task_job.py:151} INFO - Task exited with return code 0
